app/src/main/java/com/randomappsinc/studentpicker/home/MainActivity.java

```diff
@@ -137,7 +136,7 @@ public void showPleaseRateDialog() {
                 .negativeText(R.string.no_im_good)
                 .positiveText(R.string.will_rate)
                 .onPositive((dialog, which) -> {
-                    Uri uri =  Uri.parse("market://details?id=" + getApplicationContext().getPackageName());
+                    Uri uri = Uri.parse("market://details?id=" + getApplicationContext().getPackageName());
```

Comment:
Thanks for making this change! So before, I think we had to use getApplicationContext() because the code was nested within an anonymous class. Now that it's more exposed with lambda, I think we can do getPackageName() directly since we're within an activity. Try this and if it compiles, let's make the change.

```suggestion
                    Uri uri = Uri.parse("market://details?id=" + getPackageName());
```